src/org/openbravo/base/session/OBPropertiesProvider.java

```diff
@@ -99,6 +100,25 @@ public void setFormatXML(InputStream is) {
     }
   }
 
+
+  private void loadEnvProperties() {
+    log.debug("Loading environment variables into Openbravo properties");
```

Comment:
The debug message must be like "Loading enviroment variables into obProperties from Properties Provider